cmd/zetae2e/local/local.go

```diff
@@ -83,12 +86,15 @@ func NewLocalCmd() *cobra.Command {
 	cmd.Flags().Bool(flagVerbose, false, "set to true to enable verbose logging")
 	cmd.Flags().Bool(flagTestAdmin, false, "set to true to run admin tests")
 	cmd.Flags().Bool(flagTestEthStress, false, "set to true to run eth stress tests")
-	cmd.Flags().Bool(flagTestSolanaStress, false, "set to true to run solana stress tests")
+	cmd.Flags().Bool(flagTestSolanaStress, false, "set to true to run sSlana stress tests")
 	cmd.Flags().Bool(flagTestSuiStress, false, "set to true to run sui stress tests")
 	cmd.Flags().Int(flagIterations, 100, "number of iterations to run each performance test")
-	cmd.Flags().Bool(flagTestSolana, false, "set to true to run solana tests")
+	cmd.Flags().Bool(flagTestSolana, false, "set to true to run Solana tests")
 	cmd.Flags().Bool(flagTestTON, false, "set to true to run TON tests")
 	cmd.Flags().Bool(flagTestSui, false, "set to true to run Sui tests")
+	cmd.Flags().Bool(flagSetupSolana, false, "set to true to setup Solana protocol contracts during setup")
+	cmd.Flags().Bool(flagSetupSui, false, "set to true to setup Sui protocol contracts during setup")
+	cmd.Flags().Bool(flagSetupTON, false, "set to true to setup TON protocol contracts during setup")
```

Comment:
NIT : is there a reason why we use TON (all caps ) (Sui and oslana seem to follow the same pattern )